Bio/Restriction/Restriction.py

```diff
@@ -2323,14 +2323,14 @@ def _boundaries(self, start, end):
             return start, end, self._test_normal
 
     def _test_normal(self, start, end, site):
-        """Test if site is between start and end.
+        """Test if site is between start and end (PRIVATE).
 
         Internal use only
         """
         return start <= site < end
 
     def _test_reverse(self, start, end, site):
-        """Test if site is between end and start (for circular sequences).
+        """Test if site is between end and start (for circular sequences) (PRIVATE).
```

Comment:
To try to keep this line under 80 characters, how about:

``Test if site is between end and start (for circular sequences) (PRIVATE).``

-->

``Test if site between end & start (for circular sequences) (PRIVATE).``